src/openai.ts

```
@@ -0,0 +1,104 @@
+        this.ws?.send(JSON.stringify(startMessage));
+        resolve();
+      });
+
+      this.ws.on('message', (data: WebSocket.Data) => {
+        try {
+          const messageStr = data instanceof Buffer ? data.toString('utf8') : data.toString();
+          const message: TranscriptionResult = JSON.parse(messageStr);
+          // Emit transcription events for consumers of this module.
+          this.emit('transcription', message);
+        } catch (error) {
+          this.emit('error', new Error('Failed to parse message from OpenAI: ' + error));
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)